docs/arch.gv

```diff
@@ -0,0 +1,139 @@
+digraph {
```

Comment:
A rendered form of this graph can be seen here:

https://rohanpm.github.io/cdn-lambda/arch.html